packages/neuron-ui/src/widgets/Pagination/pagination.module.scss

```diff
@@ -1,7 +1,7 @@
-$primary-color: #1b73b3;
-$secondary-color: #717171;
 $shadow-color: rgba(0, 0, 0, 0.05);
-$disable-color: $secondary-color;
+$disable-button-text-color: var(--disable-button-text-color);
+$enable-button-text-color: var(--enable-button-text-color);
+$secondary-background-color: var(--secondary-background-color);
```

Comment:
CSS variables could be used directly since it's generally adopted.
Sass variables will be removed gradually

Reply:
> CSS variables could be used directly since it's generally adopted. Sass variables will be removed gradually

ok, got it.